docs/CONTRIBUTING.md

```
@@ -1,338 +1,182 @@
-    metric_2: float
+PRs that swap neutral capability descriptions for vendor marketing copy (superlatives like "fastest," "most accurate," "best-in-class," or direct taglines from a vendor's homepage) are closed without merge. Documentation describes _what a tool does_, not _how good it is_.
 
-class MyToolCalculator:
-    def __init__(self, prices: pd.DataFrame):
-        self.prices = prices
+Example of what we reject:
 
-    def calculate(self) -> MyToolOutput:
-        # Implementation
-        return MyToolOutput(metric_1=..., metric_2=...)
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)